spyder/plugins/findinfiles/plugin.py

```diff
@@ -92,14 +92,6 @@ def register(self):
                 menu=menu,
             )
 
-        if toolbar:

```

Comment:
On top of this class you can see this constant:

```python
OPTIONAL = [Plugins.Editor, Plugins.Projects, Plugins.WorkingDirectory,
            Plugins.MainMenu, Plugins.Toolbar]
```

Please remove `Plugins.Toolbar` because now it's not necessary.

Reply:
And also remove the `toolbar = self.get_plugin(Plugins.Toolbar)` variable, declared at the beginning of the `register` method.

Reply:
Done!